apps/agent/Dockerfile

```
@@ -4,19 +4,47 @@
  && bun run --filter '@aqsha/agent' build
 
 FROM node:24-slim AS runtime
 WORKDIR /app/apps/agent
 ENV NODE_ENV=production
 ENV PORT=4317
 ENV HOST=0.0.0.0
+# Infisical CLI: the entrypoint `infisical run`s the app so runtime secrets come from Infisical /app.
+# Above the commit-changing ENV below so it stays cached. Debian base → official apt repo.
+# pipefail: a failed setup-script download (curl -f) fails the build instead of being masked by bash.
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN apt-get update && apt-get install -y --no-install-recommends curl ca-certificates gnupg \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)